src/ripple/app/ledger/LedgerEntrySet.cpp

```diff
@@ -64,6 +69,7 @@ void LedgerEntrySet::swapWith (LedgerEntrySet& e)
     mSet.swap (e.mSet);
     std::swap (mParams, e.mParams);
     std::swap (mSeq, e.mSeq);
+    boost::swap (mFundingTable, e.mFundingTable);
```

Comment:
Note:  idiomatic way to call free function `swap`:
````
using std::swap;
swap (mParams, e.mParams);
swap (mSeq, e.mSeq);
swap (mFundingTable, e.mFundingTable);
````
If the idiomatic pattern for some reason isn't working, the code should have a comment explaining.